packages/m/mdb-ui-kit.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "mdb-ui-kit",
+  "filename": "js/mdb.min.js",
```

```suggestion
  "filename": "mdb.min.js",
```
If the filemap change is made, this will need updating to match :)